### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by EdwardNeuwirth**

### CODE REVIEW

**Review by franzspiess — APPROVED** [review]

File: client/src/App.js [hunk 1]
```diff
@@ -1,59 +1,54 @@
...
     return (
       <div>
-        <ContractList />
-        <ContractPage id={2} />
-        <Login />
-        <CreateNegotiation />
-      </div>);
-
+        <Dashboard />
+      </div>
```
Comment (oliversisson) [inline client/src/App.js:39]: is this <div> that wraps Dashboard even required?

File: client/src/containers/Dashboard/Dashboard.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+import Dashboard from './Dashboard';
+
+afterEach(cleanup);
+
+test('should not complain, that there is no test for now', () => {
+  expect(true).toBeTruthy();
+});
```
Comment (oliversisson) [inline client/src/containers/Dashboard/Dashboard.test.js:10]: #facepalm :stuck_out_tongue_closed_eyes:

File: client/src/containers/Dashboard/index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import Dashboard from './Dashboard';
+
+export default Dashboard;
```
Comment (oliversisson) [inline client/src/containers/Dashboard/index.js:3]: I don't see the point of these files..

**Review by oliversisson — COMMENTED** [review]
some thoughts

**Pull request merged**